tdvt/tdvt/tabquery_path.py

```diff
@@ -0,0 +1,23 @@
+class TabQueryPath(object):
+    def __init__(self, linux_path, mac_path, windows_path):
+        self.linux_path = linux_path
+        self.mac_path = mac_path
+        self.windows_path = windows_path
+
+    @staticmethod
+    def from_array(paths):
+        if len(paths) != 3:
+            raise IndexError
+        t = TabQueryPath(paths[0], paths[1], paths[2])
+        return t
+
+    def to_array(self):
+        return [self.linux_path, self.mac_path, self.windows_path]
+
+    def get_path(self, os):
```

Comment:
Do we gain anything by requiring an `os` arg? The function could just be something like
```python
def get_path(self):
   if sys.platform.startswith('darwin'):
        ...
```

Reply:
Not really. It's used for testing. I'll think about changing it.